inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/manager/WorkloadAndWorkflowServiceImplBase.java

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright 2020
+ * Ubiquitous Knowledge Processing (UKP) Lab
+ * Technische Universität Darmstadt
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.tudarmstadt.ukp.inception.workload.dynamic.manager;
+
+import java.util.List;
+
+import javax.persistence.EntityManager;
+
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+import de.tudarmstadt.ukp.clarin.webanno.api.ProjectService;
+import de.tudarmstadt.ukp.clarin.webanno.model.Project;
+
+@Component(WorkloadAndWorkflowService.SERVICE_NAME)
+public class WorkloadAndWorkflowServiceImplBase implements WorkloadAndWorkflowService
+{
+    private final EntityManager entityManager;
+    private final ProjectService projectService;
+
+    public WorkloadAndWorkflowServiceImplBase(
+        EntityManager aEntityManager, ProjectService aProjectService)
+    {
+        entityManager = aEntityManager;
+        projectService = aProjectService;
+    }
+
+    @Override
+    @Transactional
+    public String getWorkflowManager(Project aProject)
+    {
+        List<String> result = entityManager.createQuery(
+            "SELECT workflowmanager " +
+                "FROM WorkloadAndWorkflow " +
+                "WHERE project = :projectID", String.class)
+            .setParameter("projectID", aProject).getResultList();
+
+        if (result.isEmpty()) {
+            entityManager.persist(new WorkloadAndWorkflow(
+                aProject,"Default workload manager","Default monitoring page","detault",6));
```

Comment:
detault
please use constants and or default constructors

Reply:
I would define a default constructor once or at least a function here or in the DAO so that you do not duplicate the default wf manager code everywhere